### PR TITLE
app: add metric for threshold and num validators

### PR DESCRIPTION
**Pull request by xenowits**
Add metric for `threshold`, `num_validators` and `peer_name`.

category: misc
ticket: https://github.com/ObolNetwork/charon-distributed-validator-node/issues/65

### CODE REVIEW

**Comment by xenowits** [comment]
Screenshot 2022-08-31 at 2 07 55 PM

File: app/metrics.go [hunk 1]
```diff
@@ -55,11 +69,13 @@ var (
 	})
 )
 
-func initStartupMetrics(lockHash string) {
+func initStartupMetrics(lockHash string, threshold, numValidators int) {
```
Comment (corverroos) [inline app/metrics.go:72]: add one for `peer_name` as well 🙏

**Review by corverroos — APPROVED** [review]

**Comment by xenowits** [comment]
Screenshot 2022-08-31 at 6 43 27 PM

The updated simnet dashboard

**Comment by xenowits** [comment]
Screenshot 2022-08-31 at 6 58 37 PM

The update simnet dashboard.

**Pull request merged**